handlers/user.go

```diff
@@ -1431,9 +1443,15 @@ func UserConfirmUpdateEmail(w http.ResponseWriter, r *http.Request) {
 
 	err = db.FrontendWriterDB.Get(&user, "SELECT id, email, email_confirmation_ts, email_confirmed, email_change_to_value FROM users WHERE email_confirmation_hash = $1", hash)
 	if err != nil {
-		logger.Errorf("error retrieving email for confirmation_hash %v %v", hash, err)
-		utils.SetFlash(w, r, authSessionName, "Error: This confirmation link is invalid / outdated.")
-		http.Redirect(w, r, "/confirmation", http.StatusSeeOther)
+		logger.Errorf("error retrieving user data for updating email; hash: %v; err: %v", hash, err)
```

Comment:
Should use `utils.LogError`. Would require multiple changes in this file.

Reply:
Thx, changed to `utils.LogError` for all functions I touched.